README.md

```diff
@@ -6,7 +6,61 @@ For more details read the [TSC Charter](https://github.com/nodejs/TSC/blob/maste
 
 If your project is interested in joining the Node.js Foundation please read the [Project Lifecycle.md](./Project-Lifecycle.md) documentation.
 
+## TSC Scope of Responsibility
+
+**The TSC is responsible for Node.js Core project, a number of projects depended upon by Node.js Core and a number of projects adjacent to Node.js Core.**
+
+### Node.js Core
+
+Node.js Core is defined as the code project existing in the repository located at <https://github.com/nodejs/node> _minus_ the vendored projects included in the codebase, specifically those located in the [deps](https://github.com/nodejs/node/tree/master/deps) and [tools](https://github.com/nodejs/node/tree/master/tools) directories that are copied in from externally maintained repositories.
+
+Also included in the Node.js Core project are projects required to plan, test, build, document and ship Node.js releases but _are not significant enough to require independent management and administration_. These projects are located in the following repositories:
+
+* https://github.com/nodejs/docs
+* https://github.com/nodejs/testing
+* https://github.com/nodejs/node-eps
+* https://github.com/nodejs/security (private)
+
+### Dependent Projects
+
+Additional projects fall within the TSC's scope of responsibility which are dependents of Node.js Core in that they are required to plan, test, build, document and ship Node.js releases, yet _are significant enough to require independent management and administration_. These projects are located in the following repositories:
+
+* https://github.com/nodejs/nodejs.org
```

Comment:
In my ongoing quest make sure the I18N groups get recognition for their hard work, can we add something like this to the end of this line: "and all the i18n sites"